### ISSUE TITLE
Runttime errors in Google Colab on requires_grad_ is not supported on ScriptModules

### ISSUE THREAD

**Issue by metalaureate** [mfrashad/text2art#5]
Hi, thank you for sharing this repo. I wondered if you could help me get it to run in the Colab, because it does not seem run as-is. I know Python but I am a noob in this ML domain.

Off the bat it was failing on `import clipit` on a call to `torchtext` which appeared to be versioning issue. I added `!pip install -U torchtext`which then led to this failure further down the code.

```
RuntimeError                              Traceback (most recent call last)
<ipython-input-2-25d790141602> in <module>()
     12 # Apply these settings and run
     13 settings = clipit.apply_settings()
---> 14 clipit.do_init(settings)
     15 clipit.do_run(settings)

1 frames

/usr/local/lib/python3.7/dist-packages/torch/jit/_script.py in fail(self, *args, **kwargs)
    912     def _make_fail(name):
    913         def fail(self, *args, **kwargs):
--> 914             raise RuntimeError(name + " is not supported on ScriptModules")
    915 
    916         return fail

RuntimeError: requires_grad_ is not supported on ScriptModules
```

**Comment by metalaureate** [comment]
I checked all the forks of this repo and they all produce this error. I found this recent unanswered thread: https://github.com/pytorch/pytorch/issues/68340

**Comment by metalaureate** [comment]
This occurs on all of the clipit submodules and forks too, in the Colab and also on a local Notebook. Sorry I don't know enough to work out how to fix it.

**Comment by mfrashad** [comment]
Hi, this was caused by update in `torch-text`, should be fixed with `torchtext==0.10.0`

**Comment by metalaureate** [comment]
Thank you that fixed it. It does appear to conflict torchvision but I am not sure if that will matter.

**Comment by metalaureate** [comment]
hi! The Collab notebook now crashes 'for unknown reason' and restarts without error when it tries to run. Probably not your code. 

```
ov 21, 2021, 12:44:07 PM | WARNING | WARNING:root:kernel 54fa81f6-a41a-4a10-96a8-92d8afbd3955 restarted
-- | -- | --
Nov 21, 2021, 12:44:07 PM | INFO | KernelRestarter: restarting kernel (1/5), keep random ports
Nov 21, 2021, 12:28:06 PM | INFO | Adapting to protocol v5.1 for kernel 54fa81f6-a41a-4a10-96a8-92d8afbd3955
Nov 21, 2021, 12:28:04 PM | INFO | Kernel started: 54fa81f6-a41a-4a10-96a8-92d8afbd3955
```

**Comment by Unmesh28** [comment]
> Hi, this was caused by update in `torch-text`, should be fixed with `torchtext==0.10.0`

installed torchtext==0.10.0 using 
`pip install torchtext==0.10.0`

Getting error : 
`/home/ubuntu/anaconda3/envs/textToArt/lib/python3.7/site-packages/torchvision/io/image.py:11: UserWarning: Failed to load image Python extension: /home/ubuntu/anaconda3/envs/textToArt/lib/python3.7/site-packages/torchvision/image.so: undefined symbol: _ZNK2at10TensorBase21__dispatch_contiguousEN3c1012MemoryFormatE
  warn(f"Failed to load image Python extension: {e}")
Segmentation fault (core dumped)
`

**Comment by superwhyun** [comment]
I think I found a walk-around.

In the clipit.py:417,

`       perceptor = clip.load(clip_model, jit=jit)[0].eval().requires_grad_(False).to(device)
`
to
`        perceptor = clip.load(clip_model, jit=False)[0].eval().requires_grad_(False).to(device)
` 

After this slight modification, it works well in my local environment but has not been tested in colab.
Open AI's CLIP that is used in this repo does not yet support jit, I think.

p.s. I have used pytorch-1.10.2 and cuda 11.4.
